crates/flox/src/commands/environment.rs

```diff
@@ -30,15 +33,66 @@ pub struct EnvironmentArgs {
 
 pub type EnvironmentRef = String;
 
+#[derive(Bpaf, Clone)]
+pub enum EnvironmentSelect {
+    Dir(
+        /// Path containing a .flox/ directory
+        #[bpaf(long("dir"), short('d'), argument("path"))]
+        PathBuf,
+    ),
+    Remote(
+        /// A remote environment on floxhub
+        #[bpaf(long("remote"), short('r'), argument("owner/name"))]
+        environment_ref::EnvironmentRef,
+    ),
+}
+
+impl Default for EnvironmentSelect {
+    fn default() -> Self {
+        EnvironmentSelect::Dir(PathBuf::from("./"))
```

Comment:
shouldn't matter too much.
This might also be implemented by the "discover upward" method for .flox dirs and is currently also missing getting the "latest activated env".
Not sure if these can be implemented as `Default` (i.e. taking no args at all). This was the path of least resistance for now.